### DISCUSSION TITLE
Suggest about Interlace effect

### DISCUSSION THREAD

**Discussion by namp47685** [ntsc-rs/ntsc-rs#178] [Ideas]
is there any plan to add interlacing effect to NTSC-RS?
i have seen many VHS footage that has interlacing
like this capture from Wikipedia: [https://upload.wikimedia.org/wikipedia/commons/1/19/Interlaced_video_frame_%28car_wheel%29.jpg](url)

**Comment by notdigidani** [comment]
the program already has a interlacing effect, tho it doesnt apply to the video itself. set the feild to "Interleaved" and you'll se what im sayin

**Reply by bismuthdistrict** [reply]
it doesn't seem to make a difference unless you have your edge wave cranked up like all the way

**Reply by notdigidani** [reply]
thats what im sayig. the program only interlaces the VHS effect, not the video itself.
